### ISSUE TITLE
UnhandledPromiseRejectionWarning: TypeError: \npm\node_modules\jscpd\html\layout.pug:45

### ISSUE THREAD

**Issue by kentmz** [kucherenko/jscpd#214]
**Describe the bug**
This is related to the bug [#213](https://github.com/kucherenko/jscpd/issues/213). After upgrading to 2.0.12 and running the same command I now get a different issue.

**To Reproduce**
When I run the command:
jscpd D:\code -f "css" -r "html"
I get the following exception

```
jscpd D:\code -f "css" -r "html"
(node:10388) UnhandledPromiseRejectionWarning: TypeError: \npm\node_modules\jscpd\html\layout.pug:45
    43|               type: 'pie',
    44|               data: [
  > 45|                 { value: #{total.lines}, name: 'Total' },
    46|                 { value: #{total.duplicatedLines}, name: 'Copy/Pasted' },
    47|               ]
    48|             }

Cannot read property 'lines' of undefined
    at eval (eval at wrap (\npm\node_modules\jscpd\node_modules\pug-runtime\wrap.js:6:10), <anonymous>:343:63)
    at template (eval at wrap (\npm\node_modules\jscpd\node_modules\pug-runtime\wrap.js:6:10), <anonymous>:1636:127)
    at HtmlReporter.report (\npm\node_modules\jscpd\build\reporters\html.js:30:20)
    at \npm\node_modules\jscpd\build\jscpd.js:375:38
    at Array.map (<anonymous>)
    at JSCPD.<anonymous> (\npm\node_modules\jscpd\build\jscpd.js:374:77)
    at step (\npm\node_modules\jscpd\build\jscpd.js:43:23)
    at Object.next (\npm\node_modules\jscpd\build\jscpd.js:24:53)
    at fulfilled (\npm\node_modules\jscpd\build\jscpd.js:15:58)
    at <anonymous>
(node:10388) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:10388) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Also, if I use the report option as console instead of html, it just shows the Execution time and nothing else.

**Expected behavior**
It should generate the report.

**Comment by dialoguewise** [comment]
Thanks for fixing this quickly. Appreciate it. Now when I run the command with -r as "console" it just shows the execution time and nothing else. If I use "html", there is nothing in it. Please find attached report.

[report.zip](https://github.com/kucherenko/jscpd/files/3024158/report.zip)

**Comment by kucherenko** [comment]
Looks like no duplicates detected

**Comment by kentmz** [comment]
There are duplicates. Please see attached file. Also, there are two css and it only analyses one file.
Also, have a look at the attached screenshot. I ran the same command four times with the same files, sometimes it shows the output table sometimes only the execution time.

[cssAnalysis.zip](https://github.com/kucherenko/jscpd/files/3025270/cssAnalysis.zip)
![jscpd](https://user-images.githubusercontent.com/38720758/55273623-f7ec5e80-5321-11e9-88ee-6721a37dcda0.png)

**Comment by kucherenko** [comment]
ohh... thank you, will investigate the issue

**Comment by kucherenko** [comment]
fixed in jscpd@3.3.0-rc.9
